devicetypes/smartthings/zwave-water-valve.src/zwave-water-valve.groovy

```diff
@@ -59,18 +59,22 @@ def updated() {
 
 def parse(String description) {
 	log.trace "parse description : $description"
-	def result = null
 	def cmd = zwave.parse(description, [0x20: 1])
 	if (cmd) {
-		result = createEvent(zwaveEvent(cmd))
+		def events = zwaveEvent(cmd)
+		def evt1 = createEvent(events[0])
+		log.debug "Parse returned ${evt1?.descriptionText}"
+		def evt2 = createEvent(events[1])
+		log.debug "Parse returned ${evt2?.descriptionText}"
 	}
-	log.debug "Parse returned ${result?.descriptionText}"
-	return result
+	return [evt1, evt2]
```

Comment:
I think @workingmonk is saying that `parse()` can return a list of events created via `createEvent()` as it was before. I don't see any `createEvent()` API that accepts a list of event info, it only takes a single event definition via a `Map`.